lib/ws.js

```diff
@@ -40,6 +40,10 @@ class BinanceWS {
         path = (isCombined ? this._combinedBaseUrl : this._baseUrl) + path;
         const ws = new WebSocket(path);
 
+        ws.on('open', () => {
+            setInterval(() => { ws.pong() }, 60000)
+        })
```

Comment:
Thanks for the contribution! If this ws closes this has a slight risk of building a number of untracked interval timers trying to send pongs on a dead websocket. You could track the interval and use clearInterval if the websocket closes, but this becomes more complicated when there's multiple active websockets (where will you store the timer for that websocket).

Instead of sending unsolicited pongs, what I've successfully done (including with binance websockets) is spawn a monitoring abstraction on any websocket. This can send pings at intervals, handle pongs, and if disconnected it'll clean up any timers and emit that. Feel free to implement it here, as is or simplified:
https://gist.github.com/tiagosiebler/2aaae16001168b1766f7c99aa5f57402

Reply:
Hi @tiagosiebler!

Thanks for great review!
I will check the suggested solution and will try to implement it, thanks!